rtcpeerconnection.go

```diff
@@ -27,6 +29,22 @@ import (
 // comparisons when no value was defined.
 const Unknown = iota
 
+// RTCPeerLogger is the default logging struct for rtcpeerconnection.
+type RTCPeerLogger struct{
+	INFO, DEBUG, WARNING,
```

Comment:
I would make these private, otherwise there might be confusion and people could call them directly.